frontend/package.json

```diff
@@ -158,6 +159,7 @@
     "@storybook/node-logger": "^6.3.10",
     "@storybook/preset-create-react-app": "^3.2.0",
     "@storybook/react": "^6.3.10",
+    "@types/react-window": "^1.8.5",
```

Comment:
Yeah, this should go to dependencies. Anything required for `npm run build` is a dependency not a dev dependency.

Reply:
fixed